.github/workflows/check-service-labels.yml

```diff
@@ -13,8 +13,10 @@ jobs:
   check-labels:
     runs-on: ubuntu-latest
     steps:
-    - name: Check servicing labels
+    - name: Check `Servicing-approved` label
       run: |
+        echo "Merging permission is enabled for servicing PRs when the `Servicing-approved` label is applied."
+        echo "Rules for applying the label can be found [here](~/docs/project/library-servicing.md)."
         if [ "${{ contains(github.event.pull_request.labels.*.name, 'Servicing-approved') }}" = "true" ]; then
```

Comment:
@ericstj line 20 was the confusing part. The user just sees `if [ true = true ]; then` which does not tell anything. I am hoping the preceding echos will also show up.

Reply:
Echo helps, but can you instead just emit an error and control the error text?

Reply:
Sure. I think you mean an echo that prints a message prefixed with `::error`. Is that correct?

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-creating-an-annotation-for-an-error

Reply:
Done. Let's see how it looks.